web_client/panels/AnnotationSelector.js

```diff
@@ -13,7 +13,8 @@ import '../stylesheets/panels/annotationSelector.styl';
 var AnnotationSelector = Panel.extend({
     events: _.extend(Panel.prototype.events, {
         'click .h-toggle-annotation': 'toggleAnnotation',
-        'click .h-delete-annotation': 'deleteAnnotation'
+        'click .h-delete-annotation': 'deleteAnnotation',
+        'click .h-download-annotation': 'downloadAnnotation'
```

Comment:
I'm failing to understand how this works -- where is `downloadAnnotation` defined?